### PR TITLE
Fix up containers

### PR DESCRIPTION
**Pull request by marty331**
- Add command to run sub as doit localsub
- Add command to run hub as doit localhub
- Update docs to reflect the change

### CODE REVIEW

File: src/sub/app.py [hunk 1]
```diff
@@ -186,3 +186,4 @@ def after_request(response):
     app.debug = True
     app.use_reloader = True
     app.run(host="0.0.0.0", port=CFG.LOCAL_FLASK_PORT)
+    logger.info("sub running", app=app)
```
Comment (sidler-mozilla) [inline src/sub/app.py]: don't want the same thing in hub/app.py?

File: src/hub/app.py [hunk 2]
```diff
@@ -21,6 +21,7 @@
 
 logger = get_logger()
 
+
```
Comment (sidler-mozilla) [inline src/hub/app.py]: just cause?

File: dodo.py [hunk 3]
```diff
@@ -8,7 +8,9 @@
 import itertools
 import contextlib
 import threading
+import atexit
```
Comment (sidler-mozilla) [inline dodo.py]: are you using this?

Comment (marty331) [reply dodo.py]: Nope, forgot to remove.

File: src/sub/app.py (hunk 1 above)
Comment (marty331) [reply src/sub/app.py]: Yes, changing.

File: src/hub/app.py (hunk 2 above)
Comment (marty331) [reply src/hub/app.py]: Not sure what you are calling out?

**Pull request merged**